pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -56,38 +57,36 @@ var _ = Describe("Node-labeller config", func() {
 
 		clusterConfig, _, _ := testutils.NewFakeClusterConfigUsingKV(kv)
 
+		cpuCounter := &libvirtxml.CapsHostCPUCounter{
+			Name:      "tsc",
+			Frequency: 4008012000,
+			Scaling:   "no",
+		}
+
 		nlController = &NodeLabeller{
 			nodeClient:              nil,
 			clusterConfig:           clusterConfig,
 			logger:                  log.DefaultLogger(),
 			volumePath:              "testdata",
 			domCapabilitiesFileName: "virsh_domcapabilities.xml",
+			cpuCounter:              cpuCounter,
```

Comment:
Since this test isn't concerned about the CPU counter, please consider passing `nil` here.

Reply:
Good catch, done.